src/bot/listeners/test-tensor.listener.js

```diff
@@ -73,25 +73,57 @@ class TestTensorListener {
         return;
       }
 
-      if (this.storage[this.getStorageKey(ctx)].positives?.length === this.storage[this.getStorageKey(ctx)].negatives?.length) {
+      const positivesCount = this.storage[this.getStorageKey(ctx)].positives?.length;
```

Comment:
Could you create the same `const storage = this.storage[this.getStorageKey(ctx)];` here and use it instead of all `this.storage[this.getStorageKey(ctx)]` ?